### PR TITLE
refactor(app) : Integrate EVM chains

### PR DESCRIPTION
**Pull request by vnksnkr**
Implements https://app.clickup.com/t/9002019994/PRF-4899

### CODE REVIEW

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-4899 Integrate EVM chains with public key and add account](https://app.clickup.com/t/9002019994/PRF-4899)

File: .vscode/settings.json
Comment (ujjwal-cyph) [inline .vscode/settings.json:1]: let's undo. we do not maintain this file.

File: apps/evm_family/avalanche/avalanche_app.c [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+ * STATIC FUNCTION PROTOTYPES
+ *****************************************************************************/
+
+/*****************************************************************************
+ * STATIC VARIABLES
+ *****************************************************************************/
+
+static const evm_config_t avalanche_app = {
+    .lunit_name = "AVAX",
+    .name = "Avalanche CChain",
```
Comment (ujjwal-cyph) [inline apps/evm_family/avalanche/avalanche_app.c:99]: Let's keep the exact name as previous unless there is a good enough reason to change.

**Review by ujjwal-cyph — CHANGES_REQUESTED** [review]

File: apps/evm_family/avalanche/avalanche_app.c (hunk 1 above)
Comment (vnksnkr) [reply apps/evm_family/avalanche/avalanche_app.c:99]: `#define EVM_LONG_NAME_MAX_SIZE 16`

Can we increase the max size?

Comment (ujjwal-cyph) [reply apps/evm_family/avalanche/avalanche_app.c:99]: yes. definitely.

Comment (ujjwal-cyph) [reply apps/evm_family/avalanche/avalanche_app.c:99]: do ensure you consider the null character when defining the size. and ensure to update the comment too.

**Review by ujjwal-cyph — APPROVED** [review]
LGTM

**Review by amanCypherock — APPROVED** [review]

**Pull request merged**